packages/react/package.json

```diff
@@ -31,7 +31,6 @@
     "@neutrinojs/loader-merge": "^8.2.0",
     "@neutrinojs/web": "^8.2.0",
     "deepmerge": "^1.5.2",
-    "eslint": "^4.19.1",
```

Comment:
So I guess now if people don't add eslint to their `package.json` they will get peer dependency warnings from `eslint-plugin-react` (and similar for the vue/preact presets etc). However maybe that's ok and their choice to save the dependency?

The only option I can think of (other than leaving this dependency in here), is to stop the react preset from including the eslint react plugin, and then airbnb guide users can just use the airbnb preset (which does include the plugin), and for standardjs perhaps we'd have to add a second "standardjs-react" preset? (Maybe not worth it...)

Reply:
Hmm yeah, hadn't thought of this one. I think the easiest thing at the moment would be to leave it until we think of something better, if possible.